wp-auth.js

```diff
@@ -125,6 +125,63 @@ WP_Auth.prototype.getUserMeta = function (id, key, callback) {
 	);
 };
 
+
+/**
+ * Function will return object of meta_keys paired with meta_value if found
+ * 
+ * @param {*} id  User ID
+ * @param {*} keys array of meta keys
+ * @param {*} callback 
+ * @returns 
+ */
+WP_Auth.prototype.getUserMetas = function (id, keys, callback) {
+	if (!(id in this.meta_cache_timeout)) this.meta_cache_timeout[id] = {};
+	if(!keys || !keys.length) return;
+	
+	// sanitizeValue inside values and make new key for cache based on given array
```

Comment:
Let's keep it simple for now and use key bases on the array provided by the user.